lib/install/data/data.go

```diff
@@ -190,4 +190,8 @@ func (d *Data) CopyNonEmpty(src *Data) {
 		d.ContainerNetworks = src.ContainerNetworks
 	}
 	d.Timeout = src.Timeout
+	if src.HTTPProxy != nil || src.HTTPSProxy != nil {
+		d.HTTPProxy = src.HTTPProxy
```

Comment:
hmm... it's a good question.
we're taking set of configuration to replace the old set of configuration if any one of them appear in the options, for example to switch network configuration from static ip back to dhcp.
It's hard to know how user feels about that.
For this specific issue, is there anyone using different proxy server for http and https? If that's not the common case, I guess this definition is fine.